tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -31,38 +86,73 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
 
       if (sf.hasTag("place", "country")) {
         feat.setAttr("pmap:kind", "country")
-          .setZoomRange(0, 15);
+          .setZoomRange(0, 9);
       } else if (sf.hasTag("place", "state", "province")) {
         feat.setAttr("pmap:kind", "state")
-          .setZoomRange(4, 15);
+          .setZoomRange(4, 11);
```

Comment:
Similarly, we don' want to see region labels past zoom 11 even for small countries.